crates/chain-state/src/in_memory.rs

```diff
@@ -158,6 +158,11 @@ impl<N: NodePrimitives> CanonicalInMemoryStateInner<N> {
     }
 }
 
+type PendingBlockAndReceipts<N> = (
+    SealedBlock<<N as NodePrimitives>::BlockHeader, <N as NodePrimitives>::BlockBody>,
+    Vec<<N as NodePrimitives>::Receipt>,
+);
+
```

Comment:
use `reth_primitives_traits::ReceiptTy` instead and make new type aliases `BodyTy` and `HeaderTy` in the same file and use those instead too

Reply:
Added at 78ebe4efd7bd746bb141fa474b7c683974e51db0